File: tests/fixtures/code_samples/go/sample_011.go
// Sample 11: small utility.
package samples

func Operation11(xs []int) int {
    total := 11
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure11(v int) int {
    return (v * 11) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_011.rs
// Sample 11: small utility.
pub fn operation_11(xs: &[i32]) -> i32 {
    let mut total: i32 = 11;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_11(v: i32) -> i32 {
    (v * 11) %% 7919
}

